flow-typed/content.js

```diff
@@ -24,7 +24,7 @@ declare type WatchHistory = {
 declare type PlayingUri = {
```

Comment:
It's not floating right in the livestreams, but at least it's better than production's where it hijacked the livestream video instead.

<img width="368" alt="image" src="https://user-images.githubusercontent.com/64950861/178887579-618cce8d-de20-4fe2-9376-20d8723cc471.png">